static/app/views/performance/data.tsx

```diff
@@ -40,6 +40,13 @@ export const COLUMN_TITLES = [
 
 const TOKEN_KEYS_SUPPORTED_IN_LIMITED_SEARCH = ['transaction'];
 
+export const getDefaultStatsPeriod = (organization: Organization) => {
+  if (organization?.features?.includes('performance-landing-page-stats-period')) {
+    return '7d';
```

Comment:
I'm setting this to 7d for now, and then will up it to 14d once we have performance data on 7d.

Reply:
Or should I just do 14d right away? 🤔

Reply:
From slack discussion, gonna try 14d first.